src/providers/persistence/persistence.ts

```diff
@@ -26,6 +26,7 @@ export interface GiftCardMap {
 
 const Keys = {
   ADDRESS_BOOK: network => 'addressbook-' + network,
+  ORDER_ADDRESS: address => 'order-address-' + address,
```

Comment:
maybe you could use **orderAddress-** instead of **order-address**

Reply:
I changed this in the next commit.